### PR TITLE
Add comments to cirq_superstaq_example.py

### LINKED ISSUE #6: Add helpful comments to cirq_superstaq_example.py
Add guiding comments for the user to understand what each line of code does. Similar to `qiskit-example-multiple-circuits` branch in qiskit-superstaq.

### PR DESCRIPTION
**Pull request by rohansk1**
Resolves #6

### CODE REVIEW

**Comment by singular-value** [comment]
@rohansk1 is this still a draft PR?

**Comment by rohansk1** [comment]
> @rohansk1 is this still a draft PR?

Oops, no

**Review by singular-value — APPROVED** [review]
lgtm

**Review by vtomole — APPROVED** [review]
LGTM

File: examples/cirq_superstaq_example.py [hunk 1]
```diff
@@ -1,19 +1,19 @@
 import cirq
```
Comment (vtomole) [inline examples/cirq_superstaq_example.py:1]: Add doc at top of file as to what this file does. Similar to https://github.com/quantumlib/Cirq/blob/c3f9a5d6d4d539fc9de737e46e9e1d728ef27379/examples/hello_qubit.py#L1

**Review by vtomole — CHANGES_REQUESTED** [review]

**Comment by vtomole** [comment]
@dowusu-antwi I've assigned this issue to you.

File: examples/cirq_superstaq_example.py [hunk 2]
```diff
@@ -1,20 +1,35 @@
-import cirq
+"""Creates and simulates a simple circuit generating a Bell state.
 
-import cirq_superstaq
+=== EXAMPLE OUTPUT ===
```
Comment (vtomole) [inline examples/cirq_superstaq_example.py:3]: This is different from the actual output of the program here. 

- We don't print the circuit
-  I don't see `output=1100011010111001...1101` when i run this.

Comment (vtomole) [reply examples/cirq_superstaq_example.py:3]: Add `print("This is the job that's created: ", job.status())` as well.

Comment (dowusu-antwi) [reply examples/cirq_superstaq_example.py:3]: Done.

**Review by vtomole — APPROVED** [review]
LGTM

**Pull request merged**